Week2/homework/maartjes-work.js

```diff
@@ -46,16 +46,18 @@ const maartjesTasks = monday.concat(tuesday);
 const maartjesHourlyRate = 20;
 
 function computeEarnings(tasks, hourlyRate) {
-  // Replace this comment and the next line with your code
-  console.log(tasks, hourlyRate);
+  const totalEarnings = tasks
+    .map(x => x.duration / 60)
```

Comment:
When you call `.map()` on an array with `tasks`, the callback function that you pass as an argument to the `.map()` method gets a single `task` as its argument. Therefore this parameter should be called `task` and not `x` (which in algebra is equivalent for 'unknown'). If you use `x` the reader of your code must mentally map `x` to mean `task` to understand what is happening. You should minimise the burden for the reader to use mental mapping like that.